.travis.yml

```diff
@@ -21,6 +21,7 @@ env:
   # Travis build matrix will test all permutations of the build
   - TEST_DIR=pageserve
     USER_ENDPOINT=https://users-l3zyoxchqa-uc.a.run.app/v1/authorization
+    EVENTS_ENDPOINT=https://events-l3zyoxchqa-uc.a.run.app
```

Comment:
`EVENTS_ENDPOINT` matches the event**s** service, but I just noticed `USER_ENDPOINT` is missing the 's' from the user**s** service. I'm not too sure how big a change that might be to fix this, but maybe it should go in this PR or separate one depending.

Reply:
Changed to USERS_ENDPOINT for consistency in 0682de4